src/sonic-host-services/tests/procdockerstatsd_test.py

```diff
@@ -36,9 +36,13 @@ def test_convert_to_bytes(self):
             ('500MiB', 524288000),
             ('66.41MiB', 69635932),
             ('333.6MiB', 349804954),
+            ('7GB', 7000000000),
+            ('800GB', 800000000000),
```

Comment:
Suggest to also add at least one GB test case with a decimal point, following my existing pattern.